README.md

```diff
@@ -992,6 +992,17 @@ To sign the user out, use the **PlayGamesPlatform.SignOut** method.
 
 After signing out, no further API calls can be made until the user authenticates again.
 
+## Decreasing apk size
+
+It is possible to decrease the size of the Play Games Services Unity Plugin by removing code for the Play Games Services features that your game doesn’t use by using Proguard. Proguard will remove the Play Games Unity plugin code for features that are not used in your game, so your game compiles only with the code that is needed and minimizes the size impact of using Play Games Services.
```

Comment:
"so your game compiles only with the code that is needed"

We may want to change this part.